### PR TITLE
sql/postgres: fix empty enum schema

### PR DESCRIPTION
**Pull request by hectorj-thetreep**
Fix https://github.com/ariga/atlas/issues/2095

### CODE REVIEW

File: sql/postgres/migrate.go [hunk 1]
```diff
@@ -133,12 +138,14 @@ func (s *state) plan(changes []schema.Change) error {
 			return fmt.Errorf("unsupported object %T", c.O)
 		}
 		create, drop := s.createDropEnum(e)
-		s.append(&migrate.Change{
-			Source:  c,
-			Cmd:     drop,
-			Reverse: create,
-			Comment: fmt.Sprintf("drop enum type %q", e.T),
-		})
+		s.append(
```
Comment (a8m) [inline sql/postgres/migrate.go]: Revert please

**Comment by hectorj-thetreep** [comment]
Tested locally, fixes my issue.

**Review by a8m — APPROVED** [review]
Thanks for the contribution 🚀

**Comment by hectorj-thetreep** [comment]
I broke the tests and I see why. Fixing.

**Pull request merged**